book/src/csi-windows.md

```diff
@@ -0,0 +1,93 @@
+# CSI Windows Suppoort
+
+## Status
+
+Status | Min K8s Version | Max K8s Version
+--|--|--
+Alpha | 1.18 | -
+Beta | 1.19 | -
+
+## Overview
+
+CSI drivers (e.g. AzureDisk, GCE PD, etc.) are recommended to be deployed as containers. CSI driver’s node plugin typically runs on every worker node in the cluster (as a DaemonSet). Node plugin containers need to run with elevated privileges to perform storage related operations. However, Windows currently does not support privileged containers. To solve this problem, [CSI Proxy](https://github.com/kubernetes-csi/csi-proxy) - that runs on the Windows host and executes a set of privileged storage operations on Windows nodes on behalf of containers in a CSI Node plugin daemonset. This enables multiple CSI Node plugin authors to execute privileges storage operations on Windows nodes without having to ship a custom privileged operation proxy (until privileged operations get enabled for containers in Windows in the future).
```

Comment:
It would be good to mention somewhere that csi controller level operations/sidecars are not supported on Windows.

Reply:
"To solve this problem, CSI Proxy is a binary that runs on the Windows host"

Reply:
udpated